packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -122,36 +123,50 @@ class MethodWebsocketRequestHandler {
     WebSocket webSocket,
     OpenMethodStreamCommand message,
   ) async {
-    // Validate targeted endpoint method
-    var endpointConnector =
-        server.endpoints.getConnectorByName(message.endpoint);
-    if (endpointConnector == null) {
+    late MethodStreamSession session;
+
+    MethodStreamCallContext methodStreamCallContext;
+    try {
+      methodStreamCallContext =
+          await server.endpoints.getMethodStreamCallContext(
+        createSessionCallback: (connector) {
+          session = MethodStreamSession(
```

Comment:
Suggestion: Add a close of this session in a `finally` statement to the try-catch.

This session is left open if we for example get a MethodNotFoundException right now.
We should probably also use a nullable session, like we did in the method call case, to get around the `late` initializion.

Reply:
Good catch 🙏 🌟 [c7c0a55](https://github.com/serverpod/serverpod/pull/2676/commits/c7c0a55585868c1e95d317e8896b8d42059076f1)